src/TCDev.APIGenerator.Schema/Attributes/CachableAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TCDev.APIGenerator.Events
+{
+
+    [AttributeUsage(AttributeTargets.Class)]
+    public class CachableAttribute : Attribute
+    {
+
+        public string cacheKey;
+        public int defaultLifeTime;
+
+        public CachableAttribute(
+            string cacheKey = "",
+            int defaultLifeTime = 60
+            )
+        {
+            this.cacheKey = cacheKey;
+            this.defaultLifeTime = defaultLifeTime;
+        }
+    }
+
+
+
```

Comment:
please remove spare empty lines

Reply:
Good hint, yep! Linting is broken apparently...gotta fix this. Thanks!